gemfiles/Gemfile.rubocop-next

```diff
@@ -2,7 +2,7 @@
 
 eval_gemfile "../Gemfile"
 
-gem "rubocop", ">= 0.87", "< 0.89"
+gem "rubocop", ">= 0.87"
```

Comment:
Could consider also making this `>= 0.89`, but it's not necessary because `rubocop-shopify` will force the necessary version.

Reply:
I think I'd like to keep it at `>= 0.87`, as that is the version we're checking for with the explicit version checks in the code.